tools/ci_connector_ops/ci_connector_ops/pipelines/pipelines/metadata.py

```diff
@@ -132,3 +156,42 @@ async def run_metadata_orchestrator_test_pipeline(
             metadata_pipeline_context.test_report = TestReport(pipeline_context=metadata_pipeline_context, steps_results=[result])
 
     return metadata_pipeline_context.test_report.success
+
+
+async def run_metadata_orchestrator_deploy_pipeline(
+    is_local: bool,
+    git_branch: str,
+    git_revision: str,
+    gha_workflow_run_url: Optional[str],
+    pipeline_start_timestamp: Optional[int],
+    ci_context: Optional[str],
+) -> bool:
+    # Get the DAGSTER_CLOUD_API_TOKEN from the environment
+    dagster_cloud_api_token = os.environ.get("DAGSTER_CLOUD_API_TOKEN")
+    if not dagster_cloud_api_token:
+        raise ValueError("DAGSTER_CLOUD_API_TOKEN not found in environment")
+
+    metadata_pipeline_context = PipelineContext(
+        pipeline_name="Metadata Service Orchestrator Unit Test Pipeline",
+        is_local=is_local,
+        git_branch=git_branch,
+        git_revision=git_revision,
+        gha_workflow_run_url=gha_workflow_run_url,
+        pipeline_start_timestamp=pipeline_start_timestamp,
+        ci_context=ci_context,
+    )
+
+    async with dagger.Connection(DAGGER_CONFIG) as dagger_client:
+        metadata_pipeline_context.dagger_client = dagger_client.pipeline(metadata_pipeline_context.pipeline_name)
+
+        async with metadata_pipeline_context:
+            deploy_orch_step = DeployOrchestratorStep(
+                context=metadata_pipeline_context,
+                title="Deploy Metadata Orchestrator to Dagster Cloud",
+                parent_dir_path=METADATA_DIR,
+                module_path=METADATA_ORCHESTRATOR_MODULE_PATH,
```

Comment:
Same as previous comment: these parameters are static and can be declared at with class variable constants.